aws-serverless-java-container-core/src/main/java/com/amazonaws/serverless/proxy/model/AwsProxyResponse.java

```diff
@@ -16,6 +16,7 @@
 import com.fasterxml.jackson.annotation.JsonInclude;
 import com.fasterxml.jackson.annotation.JsonProperty;
 
+import java.util.List;
```

Comment:
`AwsProxyResponse` should be moved to `src/test/java`

Reply:
Done